### DISCUSSION TITLE
讀書靠「智力」還是「努力」？

### DISCUSSION THREAD

**Discussion by ALAN30359704** [1101-GS4719-C/Python-Community-GS4719-C#1245] [General]
大家平常說的聰明，有一部分指的是學習效率好，也就是方法好，另一部份指的是智力高，任何東西一學就會，這兩者常被混在一起。
由於研究人類學習領域、大學入學制度多年，加上目前在臺灣大學擔任教職，我常常受邀到各級學校演講，談學習方法和入學制度的應對，對象有時候是老師，有時候是學生。而在演講中我都會問他們：讀書靠的究竟是「智力」還是「努力」呢？通常聽眾們的答案是一半一半，但其實在我看來，這題的正確答案是：這兩個都相對不重要，最重要的是「方法」。

在解釋為何是方法最重要之前，我先定義一下什麼叫做方法。我講的方法其實就是你「如何」讀書，舉凡你背英文單字是把一週的份分成 7 天背，還是一天背一週的份連續做 7 天；或是你讀數學教科書時，花多少時間在看定義等等，這些看似很小的細節都是我所謂的方法。讀書要進步，第一步就是搞清楚自己目前究竟是在用什麼方法。我的經驗是，很多學生光是要把自己目前「如何讀書」的詳細方法寫下來或講出來，就已經非常辛苦了，因為他們平常多半沒有在思考自己的讀書方法。
那麼為什麼這些細節這麼重要呢？經濟學中有一個概念叫做「生產函數」，講的是投入（原料、勞力、土地）與產出（最終產品）的關係，而這個觀念剛好可以套用在學習上。學習時，我們的投入主要是讀書時間（努力）以及腦力（智力），產出就是我們學習的表現。這也是經濟學中經典的 Cobb-Douglas 生產函數。在這個式子中，大部分人覺得比較「重要」的智力與努力，的確對表現有正面的影響，但是它的邊際效果是遞減的；或著換成白話文說，就是當你已經很努力或是智力很高時，再多努力一點或是聰明一點，對增加表現的效果其實不大。

以我們的日常經驗來說，就是完全不讀書的人多讀一個小時通常會進步不少，但已經讀了 10 個小時的人再多讀一個小時的進步空間就有限了；而智力 100 與 110 的人的平均表現差距，也絕對比智力 160 與 170 的人的差距來得更大。但是方法（在生產函數中為生產力或效率，實證研究發現其變動就是經濟成長的最大因素）在上面的式子中是不會邊際遞減的，也就是說如果你的方法進步了 10%，那不管你現在的方法已經多好了，這 10% 還是會一比一地反應在你的表現上。另外，人類的努力總有個上限，再努力的人一天最多也只能讀 24 小時，而智力雖然不是完全不會變，但能變化的幅度也很有限，而且短期內很難看到改變。但方法的精進相較之下是短期可見成效，又永無止盡的。

通常講到這裡，底下已經會有些聽眾聽不下去想要反駁我了。其中最常見的反應是：這跟他的經驗不合，他看到學習表現好的人都是很聰明（或很努力）的人。

但這樣的誤解其實是有原因的。首先，目前我觀察到的多數臺灣學生（包括臺大學生），是單純靠自己的智力與努力在讀書，沒怎麼在思考方法的。如果大多數人都是這樣，那自然看不出來方法的重要。再者，大家平常說的聰明，有一部分指的是學習效率好，也就是方法好，另一部份指的是智力高，任何東西一學就會，這兩者常被混在一起。況且，我也不是說努力和智力完全不重要，只是如果你已經花了相當時間讀書，再加上你短時間很難改變智力，那你要改變的顯然就是方法了。
所以，要如何改善讀書方法？

讀到這裡，希望你已經被我說服方法是最關鍵的因素了。如果你能找到一個更好的方法，就能夠提升你的表現，而且還有可能在短時間內看到顯著的進步。但要如何找到更好的方法呢？

如同我前面所述，第一步就是先「搞清楚現況」，知道自己目前的讀書方法是什麼。因為如果你根本不知道你現在是怎麼讀的，也不會知道可以改變的地方在那裡。第二步是「找到新方法」，這個新的方法可以自己想，也可以問別人，或是看看市面上談讀書方法的書。第三步是「實驗」，就是把自己舊的讀書方法中，你覺得效率最差的部份換成新的方法，實驗看看學習效果有沒有改善，如果有就找下一個需要改善的讀書方法；如果沒有就換一個新方法試試看。

上面這個過程一開始會有點辛苦，又花時間，但多做幾次之後，你也會對自己更加的了解，更知道什麼是需要改進的，又有哪些方法是好的，也就越來越順利了。我自己是在高一下到高二上這一年，花了整整一年時間提升了我讀數學的方法，但只再花了另外一年就把自己英文、歷史、地理等所有其他科目的讀書方法都進化了一遍。

最後，我想要強調的是，所有做人類學習的研究者都會告訴你，學習方法是個人化的，每個人的方法都不會完全一樣，因此最好的方法也不會完全相同（但是相同處通常很多）。因此，我們需要透過不斷地試驗與調整，來找到屬於自己的最佳學習方法。這樣，我們才能夠真正發揮自己的潛力，並取得更好的表現。

**Comment by GuanYann** [comment]
方法在學習過程中非常重要，因為一個好的學習方法可以提高效率、加強記憶和理解力。這些方法包括有效地時間管理、制定明確的學習目標、從不同來源和角度學習、做好筆記、及時回顧等等。而學習方法的適用性也因人而異，每個人都需要找到適合自己的方法。因此，在讀書的過程中，智力和努力固然重要，但採用的學習方法也非常關鍵。

**Comment by alicia0725** [comment]
讀書方法很需要慢慢摸索，對我自己而言，高中適合我的方法到了大學的專業科目反倒不一定適用了。但我也會因為覺得還沒有適合自己的方法，反而當作拖延的藉口不去嘗試。所以我認為無論自己是否已經有明確的方法，都還是可以先開始嘗試讀書。方法固然重要，但該有的努力還是要付出的。

**Comment by harrycoconut** [comment]
以我而言，智力真的是大於努力太多了，你的智力越高，你越知道怎麼去選對的方向，若是只有努力的話，努力錯方向只會離目標越離越遠，然而，若是智力能加上努力。這世界就沒什麼能擋得了你了😌

**Comment by Jamesmoneya** [comment]
智力真的必努力重要很多如果以數學的角度來看，努力學習知識是用加法，但是如果你的智商很高那你學到的東西應該會成指數的成長，你學得越多獲取知識的速度會更快，因為你有更多背景知識與學習的方法，但是大家還是都要努力，如果沒有智商又不努力，真的會存活不下去

**Comment by hardtagottaname** [comment]
我覺得讀書其實比較看天賦。有些人就是比較容易融會貫通，僅僅根據課本上的描述，就能舉一反三，但有些人無論跟他講多少次，不會還是不會。但我認為不可否認的是，努力也很重要，特別是一個人又有讀書天賦又努力時，想追上他的腳步簡直難如登天。

**Comment by JessyCheng2003** [comment]
我覺得讀書是要「用對方法」和「努力」，智商高但不付出時間讀書和理解，最後也不會有好結果。還有，我覺得很少事是努力達成不到的，如果還沒達到，可能只是因為還不夠努力🤣

**Comment by chia-kai-chang** [comment]
@ALAN30359704  直接把別人文章 **複製貼上** ，沒有加註任何原文資訊，也沒有說明自己的看法觀點。這樣非常不恰當。

**Comment by Alexgns** [comment]
努力能彌補智力上的不足，也能超越純粹的天賦。即使智力較低的人，通過不懈努力和有效的學習方法，仍然可以取得卓越的成績。因此，智力和努力之間的結合是最理想的學習方式。靠智力入門，但靠努力提升和實現潛力。
